src/codegen/cpu.rs

```diff
@@ -627,6 +675,7 @@ impl<'a> X86CodeGenerator<'a> {
                     reinterpret,
                 )
             }
+            CBuffer::Ptr { .. } => todo!(),
```

Comment:
@samkaufman Since the smoke-test failure is caused by this line, I am not sure how I can fix this.

Reply:
Fixed in https://github.com/samkaufman/morello/pull/10/commits/2c95c3a39b8d721bdb72b63f24ca1edf21bda008.